### ISSUE TITLE
List schedules API endpoint type doesn't match returned value

### ISSUE THREAD

**Issue by jasonsilberman** [upstash/sdk-qstash-ts#26]
Hi there,

I'm trying to list the jobs I currently have scheduled using the client API like so:

```ts
import { Client } from "@upstash/qstash";

const client = new Client({ token: process.env.QSTASH_TOKEN });

export async function listScheduledJobs() {
  return client.schedules.list();
}
```

The returned response looks like this (slightly redacted):
```json
[
  {
    "scheduleId": "scd_XXXXXX",
    "cron": "0 0 * * *",
    "createdAt": 1674415617650,
    "content": {
      "header": null,
      "body": null
    },
    "destination": {
      "type": "url",
      "url": "https://example.com/messaging-endpoint"
    },
    "settings": {
      "retries": 5
    }
  }
]
```

This matches the return type described in your [API Docs](https://docs.upstash.com/qstash/openapi#tag/Schedules/operation/listSchedules), however, it does not match the Typescript types defined in this package.
```ts
export type Schedule = {
  scheduled: string;
  cron: string;
  createdAt: number;
  content: {
    header: Record<string, string[]>;
    body: string;
  };
  destination: {
    type: "topic";
    topicId: string;
    url?: never;
  } | {
    type: "url";
    topicd?: never;
    url: string;
  };
  settings: {
    deadline?: number;
    notBefore?: number;
    retries?: number;
  };
};
```

The Typescript object is missing `scheduleId` and has a `scheduled` property that is missing in the returned response. Additionally, neither of the properties within `content` are defined as optional or possibly `null`, even though the API can return `null`.

It would be great if you could update the types defined in this package to match your API, otherwise I've got to do some not-so-nice overwriting of the types on my end.

Thanks!

**Comment by chronark** [comment]
👍 I'll do it tomorrow on monday, thanks for reporting this

**Comment by chronark** [comment]
@jasonsilberman fixed it in `v0.3.4`
